pkg/controller/drivelvgcontroller/drivelvgcontroller.go

```diff
@@ -0,0 +1,378 @@
+package drivelvgcontroller
```

Comment:
name `drivelvgcontroller` sounds confusing. Can we have `capacitycontroller` instead?

Reply:
Renamed